### ISSUE TITLE
space following "<?>" is removed

### ISSUE THREAD

**Issue by griotspeak** [nicklockwood/SwiftFormat#112]
The space following "<?>" is removed when swiftLint is run . The removal of the space forces Swift to misinterpret the symbol and newly connected expression following the symbol.

```
public class Container {
    let value: Int
    let label: String?
    init(value: Int, label: String?) {
        self.value = value
        self.label = label
    }

    func == (_ lhs: Container, _ rhs: Container) -> Container {
        return Container(lhs.value + rhs.value)
    }
}

infix operator<?>

public func<?>(label: String, r: Container) -> Container {
    return Container(value: r.value, label: label)
}

func test() {
    let one = Container(1, nil)
    let labelled = "one" <?> (one + Container(value: 1, nil)) // This line
}

```

**Comment by nicklockwood** [comment]
Thanks - I believe I've identified the problem.

**Comment by nicklockwood** [comment]
Fixed in 0.27.0
